packages/itwinui-react/src/core/Popover/Popover.tsx

```diff
@@ -98,32 +107,76 @@ type PopoverInternalProps = {
     animationFrame?: boolean;
     layoutShift?: boolean;
   };
-  /**
-   * By default, the popover will only open on click.
-   * `hover` and `focus` can be manually specified as triggers.
-   */
-  trigger?: Partial<Record<'hover' | 'click' | 'focus', boolean>>;
   role?: 'dialog' | 'menu' | 'listbox';
   /**
    * Whether the popover should match the width of the trigger.
    */
   matchWidth?: boolean;
+} & PopoverInteractionProps &
+  Record<string, any>;
```

Comment:
nit: This `any` was meant to be a placeholder during our pair programming session. Can be replaced with the "correct" type now (if possible).

Reply:
Replaced with proper type (https://github.com/iTwin/iTwinUI/pull/1942/commits/99f3b3a580c8aaffb556f85b6c331d32e72a7dad).

Reply:
I meant proper types for all arbitrary props (i.e. `UseFloatingProps`), rather than explicitly listing `nodeId`.

Reply:
Even though `usePopover` is private, I still thought it would be good to keep the parameters list small. I thought that we can add more props as and when we need them.

But I can add all `useFloating`'s props (`UseFloatingProps`) now too (https://github.com/iTwin/iTwinUI/pull/1942/commits/82850b197e18f1b7386fc73c62705498a6284187).